installer/charts/rhtap-dh/templates/app-config-content.yaml

```diff
@@ -32,6 +34,11 @@ argocd:
 artifactory:
   uiUrl: ${ARTIFACTORY__URL}
 {{- end }}
+{{- if $azureSecretObj }}
```

Comment:
nit: please move that section after `auth` to keep everything sorted alphabetically.